staging/src/k8s.io/api/admission/install/install.go

```diff
@@ -0,0 +1,31 @@
+package install
+
+import (
+	"k8s.io/api/admission/v1alpha1"
+	"k8s.io/apimachinery/pkg/apimachinery/announced"
+	"k8s.io/apimachinery/pkg/apimachinery/registered"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// InstallWithInternal registers the API group with internal types and adds types to a scheme
+func InstallWithInternal(groupFactoryRegistry announced.APIGroupFactoryRegistry, registry *registered.APIRegistrationManager, scheme *runtime.Scheme, addInternal announced.SchemeFunc) {
```

Comment:
You explained why this is the way it is (you can only register once), but at some point it would still be nice to be able to simply add.

@lavalamp are you planning to finish collapsing us onto the `APIGroupFactoryRegistry` you started last summer?

Reply:
Not sure if I'll ever get back to it, but this looks like it at least wouldn't make my life harder if I did.

Reply:
Add a comment saying it's for kubernetes internal use, we don't expect users to call this function?

Reply:
done